components/02-molecules/meta/event-meta/_yds-event-meta.scss

```diff
@@ -9,6 +9,12 @@
 
 .event-meta__date {
   @include tokens.body-xl;
+
+  .date-time {
```

Comment:
Two things jumped out to me in this file:
1. There is a class nested inside another class. That's generally a no-no. Sometimes, it's unavoidable, but should definitely set off alarm bells when you see it.
2. You're writing this code twice (in this file, and the `_yds-basic-meta.scss` file. Twice is just a warning, if I saw it in three places, that would be an "error" in my books.

And luckily, both of those concerns will be easy to fix here 😄 

Looking at this image of the inspector, you'll see that the `font-variant: small-caps` was already being applied to both of these, and in both cases, it's coming from the same file `_yds-date-time.scss`. So, I think you should be able to move this css all into that one file, and both instances (as well as future date implementations) will get the new styles.
![Screen Shot 2022-08-19 at 2 52 51 PM](https://user-images.githubusercontent.com/1663810/185696973-3f298c1f-e884-4015-9414-d9d1ac9e4966.png)
 
Give that a shot, and let me know if you run into any issues.